server_handlers.go

```diff
@@ -56,6 +63,7 @@ func serverHandshakeHandler(c *Conn) error {
 				}
 			}
 
+			c.log.Trace("Moving to flight2")
```

Comment:
Can you generalize this log message, just do a `trace` inside `flight.go` itself

Reply:
Good idea yes :+1: